### PR TITLE
WT-7069 Adding column store support to tests

### PR DESCRIPTION
**Pull request by clarissecheah**

### CODE REVIEW

File: test/suite/test_hs18.py [hunk 1]
```diff
@@ -36,6 +37,11 @@ def timestamp_str(t):
 class test_hs18(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=5MB,eviction=(threads_max=1)'
     session_config = 'isolation=snapshot'
+    key_format_values = [
+        # ('column', dict(key_format='r')),
```
Comment (sulabhM) [inline test/suite/test_hs18.py:41]: We haven't really changed this test yet, though we have disabled column store scenario, keys used are still string/row-store. Please change the keys to be scenario dependant

**Review by sulabhM — CHANGES_REQUESTED** [review]
I have left some comments.

I am wondering if we should add a scenario "key_format='i' " to be consistent between examples, where an integer type is missing.

File: test/suite/test_hs04.py [hunk 2]
```diff
@@ -54,8 +54,13 @@ class test_hs04(wttest.WiredTigerTestCase):
         ('too-low', dict(reconfig_file_max='99MB', reconfig_stat_val=None)),
         ('zero', dict(reconfig_file_max='0', reconfig_stat_val=0))
     ]
+    key_format_values = [
```
[the remark this answers is not in the document]
Comment (Siddhartha8899) [reply test/suite/test_hs04.py:57]: I am a bit confused, are you suggesting that we get rid of the code that we added to this test as it doesn't do any I/O?

[the remark this answers is not in the document]
Comment (sulabhM) [reply test/suite/test_hs04.py:57]: Yes, this test just reconfigures HS file size and doesn't actually do any table creation (row or column), so we can just leave it as it is.

[the remark this answers is not in the document]
Comment (Sean04) [reply test/suite/test_hs04.py:57]: So should we remove the column store support we added?

[the remark this answers is not in the document]
Comment (sulabhM) [reply test/suite/test_hs04.py:57]: Yes, you can revert changes to this test.

[the remark this answers is not in the document]
Comment (Sean04) [reply test/suite/test_hs04.py:57]: Okay will do

File: test/suite/test_hs09.py [hunk 3]
```diff
@@ -55,6 +55,7 @@ def create_key(self, i):
         return i
 
     def check_ckpt_hs(self, expected_data_value, expected_hs_value, expected_hs_start_ts, expected_hs_stop_ts):
+        self.session.breakpoint()
```
[the remark this answers is not in the document]
Comment (Sean04) [reply test/suite/test_hs09.py:58]: Ah yeah sorry we'll just remove that, I was trying to find where the test was failing with gdb

File: test/suite/test_hs18.py (hunk 1 above)
Comment (Sean04) [reply test/suite/test_hs18.py:41]: I think we should remove the added make scenarios from this PR and deal with that in the individual tickets.

Comment (sulabhM) [reply test/suite/test_hs18.py]: Cool, lets do that, lets remove the scenarios from the tests that are still disabled.

File: test/suite/test_hs04.py [hunk 4]
```diff
@@ -71,8 +71,14 @@ def get_stat(self, stat):
         stat_cursor.close()
         return val
 
+    def create_key(self, i):
```
Comment (kommiharibabu) [inline test/suite/test_hs04.py:74]: Why this change is done in this function, I don't see it is used anywhere in it?

File: test/suite/test_hs12.py [hunk 5]
```diff
@@ -36,10 +37,15 @@ def timestamp_str(t):
 class test_hs12(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=2MB,statistics=(all),eviction=(threads_max=1)'
     session_config = 'isolation=snapshot'
+    key_format_values = [
+        # ('column', dict(key_format='r')),
```
Comment (kommiharibabu) [inline test/suite/test_hs12.py:41]: Can you add a similar comment that is present in test_hs09.py to remember to uncomment them when it is supported?

File: test/suite/test_hs13.py [hunk 6]
```diff
@@ -35,10 +37,15 @@ def timestamp_str(t):
 class test_hs13(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=2MB,statistics=(all),eviction=(threads_max=1)'
     session_config = 'isolation=snapshot'
+    key_format_values = [
+        # ('column', dict(key_format='r')),
```
Comment (kommiharibabu) [inline test/suite/test_hs13.py:41]: same here also.

File: test/suite/test_hs12.py (hunk 5 above)
Comment (Siddhartha8899) [reply test/suite/test_hs12.py:41]: The test seems to be working now after adding (uncommenting) the Key format as 'r'.

File: test/suite/test_hs09.py [hunk 7]
```diff
@@ -43,7 +43,7 @@ class test_hs09(wttest.WiredTigerTestCase):
     uri = "table:test_hs09"
     key_format_values = [
         # The commented columnar tests needs to be enabled once columnar page instantiated is fixed in (WT-6061).
-        #('column', dict(key_format='r')),
+        # ('column', dict(key_format='r')),
```
[the remark this answers is not in the document]
Comment (sulabhM) [reply test/suite/test_hs09.py:46]: Well, actually I ran hs09 with only `test_prepared_updates_not_written_to_hs` disabled, and it runs fine:

```
diff --git a/test/suite/test_hs09.py b/test/suite/test_hs09.py
index d9943b29f..8e0588e30 100644
--- a/test/suite/test_hs09.py
+++ b/test/suite/test_hs09.py
@@ -43,7 +43,7 @@ class test_hs09(wttest.WiredTigerTestCase):
     uri = "table:test_hs09"
     key_format_values = [
         # The commented columnar tests needs to be enabled once columnar page instantiated is fixed in (WT-6061).
-        # ('column', dict(key_format='r')),
+        ('column', dict(key_format='r')),
         ('integer', dict(key_format='i')),
         ('string', dict(key_format='S')),
     ]
@@ -108,6 +108,9 @@ class test_hs09(wttest.WiredTigerTestCase):
         self.check_ckpt_hs(value2, value1, 2, 3)
 
     def test_prepared_updates_not_written_to_hs(self):
+        if self.key_format == 'r':
+            return
+
         # Create a small table.
         create_params = 'key_format={},value_format=S'.format(self.key_format)
         self.session.create(self.uri, create_params)
```

**Review by sulabhM — CHANGES_REQUESTED** [review]

File: test/suite/test_hs09.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (mm-ng) [reply test/suite/test_hs09.py:46]: We disabled this because the current branch was supposed to be for tests that run successfully in full. We are making the changes to hs09 elsewhere (enabling column store and disabling the prepared update).

**Comment by sulabhM** [comment]
@hbokhari , your change (https://github.com/wiredtiger/wiredtiger/pull/6265) could potentially conflict with this PR, please work with @Siddhartha8899 to manage any conflicting changes to hs09.

File: test/suite/test_hs09.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (sulabhM) [reply test/suite/test_hs09.py:46]: hs06 has similar disable just prepare portions, hence I thought we are enabling as much as we can

Comment (sulabhM) [inline test/suite/test_hs09.py:46]: It is still disabled for the whole test

**Review by sulabhM — APPROVED** [review]

**Review by kommiharibabu — APPROVED** [review]
LGTM
